schema/object_identification.js

```diff
@@ -43,6 +43,7 @@ const SupportedTypes = {
     "./home/home_page_artist_module",
     "./me",
     "./me/conversation",
+    "./me/conversation/invoice",
```

Comment:
Related?

Reply:
Not at all, but a slight change needed in a previous PR: https://github.com/artsy/metaphysics/pull/761/files#diff-fb938f29c2e8b607035989a2f664ea02R27